### PR TITLE
Initial draft of memcached client/needed library updates

### PR DESCRIPTION
**Pull request by jshore1296**
This change is [Reviewable](https://reviewable.io/reviews/pendo-io/appwrap/127)

### CODE REVIEW

**Review by mkj-pendo — APPROVED** [review]
:lgtm:

**Comment by mkj-pendo** [comment]
We'll depend on our repaired version of gomemcache for now; can revert to our upstream if it is ever fixed.

**Comment by mkj-pendo** [comment]
[merge]

**Comment by pendo-ops** [comment]
Branch memcached deleted

File: memorystore_test.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// +build memorystore
```
Comment (brendanwalters) [inline memorystore_test.go:1]: Differs from memorystore.go -- should probably also be !memcached

**Pull request merged**